src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.TaskList;
+
+/**
+ * Represents a command that exits the chat bot.
+ */
+public class ByeCommand implements Command {
+
+    /**
+     * Returns a boolean value to signal the bot to exit.
+     * @return True if command signals bot to be terminated.
+     */
+    public boolean shouldExit() {
```

Comment:
According to the Java coding standard for boolean variables/methods, could it be more appropriate to use isExit()?